src/pages/contributors/subcomponents/ContributorsDetails.tsx

```diff
@@ -16,7 +16,7 @@ class ContributorsDetails extends React.Component {
             The <i>Source Academy</i> is designed by and for students of the National University of
             Singapore. Students who completed the CS1101S module come back to coach their juniors as
             "Avengers" or to further develop and improve the Academy. This page includes all
-            developers who contributed to the Source Academy <i>Knight</i> (2020) and its its
+            developers who contributed to the Source Academy <i>Knight</i> (2020-21) and its its
```

Comment:
Why is this in this PR?

Reply:
This was probably in there because I branched off my teammate, who was in-charge of making that change. It should be removed here: 2601bc5.